test/testing/test_launch_performance_metrics.py

```diff
@@ -15,7 +15,7 @@ def setup(self):
 
         self.folderPath = self.testPath / '__test_files__'/'__test_demo_files__'
 
-        self.prediction = self.folderPath/ 'AxonDeepSeg_seg-axonmyelin.png'
+        self.prediction = self.folderPath/ 'image_seg-axonmyelin.png'
```

Comment:
This filename is still harcoded, you probably want to use "axonmyelin_suffix" here as well.

Reply:
Thanks for the catch. Fixed in commit  25f7dde